gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixJobScheduler.java

```diff
@@ -110,14 +113,16 @@ public class GobblinHelixJobScheduler extends JobScheduler implements StandardMe
 
   private boolean startServicesCompleted;
   private final long helixJobStopTimeoutMillis;
+  private final Duration throttleTimeoutDuration;
+  private ConcurrentHashMap<String, Instant> jobStartTimeMap;
 
   public GobblinHelixJobScheduler(Config sysConfig,
-                                  HelixManager jobHelixManager,
-                                  Optional<HelixManager> taskDriverHelixManager,
-                                  EventBus eventBus,
-                                  Path appWorkDir, List<? extends Tag<?>> metadataTags,
-                                  SchedulerService schedulerService,
-                                  MutableJobCatalog jobCatalog) throws Exception {
+      HelixManager jobHelixManager,
```

Comment:
Let's revert random whitespace changes

Reply:
In general, the autoformatter we use is https://gobblin.apache.org/docs/developer-guide/CodingStyle/